website/docusaurus.config.js

```diff
@@ -0,0 +1,105 @@
+// site configuration options.
+
+module.exports = {
+  presets: [
+    [
+      '@docusaurus/preset-classic',
+      {
+        docs: {
+          path: '../docs',
+          sidebarPath: require.resolve('./sidebars.json'),
+          routeBasePath: ''
+        },
+        theme: {
+          customCss: [
+            require.resolve('./src/css/custom.css'),
+            require.resolve('./src/css/monokai.css')
```

Comment:
This is the cause of build error >.<

It does not accept array.
https://github.com/facebook/docusaurus/blob/72792a1e5ce19626a591794dc02446749b0193fe/packages/docusaurus-theme-classic/src/index.js#L20

I spent 1 hr + to debug :(

Reply:
- how come the css works then?
- how to fix this?

Reply:
1. how come the css works then?

It doesn't work perfectly. Its probably some weird and unwanted behavior of webpack or some caching. At least in my side the TOC couldnt appear, and refresh sometimes cause blank page
I've mentioned on Discord that its generating require([])

```js
export default [
  require("infima/dist/css/default/default.css"),
  require(["/mnt/c/Users/endij/Desktop/Linux/redux-starter-kit/my-website/src/css/custom.css","/mnt/c/Users/endij/Desktop/Linux/redux-starter-kit/my-website/src/css/monokai.css"]),
];
```

2. Don't use array
- Either convert the css to theme (recommended since 2kbs on home page although small is still unused css)
- From custom.css import the monokai css
- Copy monokai css to custom css
- Put it in static folder, and use `stylesheets` field in siteconfig
- Patch docusaurus to support array in theme-classic customCss options
- Many more ....

Reply:
lol forgive me, updated

Reply:
Anyway i fixed it :( its very easy to generate the prism theme.js

Thanks for the hard work